client/css/bootstrap.css

```diff
@@ -1022,6 +1022,127 @@ tbody.collapse.in {
   -o-transition: height 0.35s ease;
   transition: height 0.35s ease;
 }
+.dropdown {
```

Comment:
I really want to be pedantic and point out that it's actually a drop-up, not a dropdown.